fip-6.md

```diff
@@ -0,0 +1,141 @@
+---
+fip: 6
+title: Provide ability to deactivate FIO Addresses and Domains
+status: Draft
+type: Functionality
+author: Casey Gardiner <casey@dapix.io>
+created: 2020-04-22
+updated: 
+---
+
+## Abstract
+This FIP implements the following:
+* Adds the ability for FIO Address owners the ability to deactivate stale or unused addresses.
+* Adds the ability for FIO Domain owners the ability to set their domain as expired.
+* Adds new API end points for FIO Domain deactivation and FIO Address burn. 
+* Adds new fees for FIO Domain deactivation and FIO Address burn.
+
+## Motivation
```

Comment:
Should add motivation for "ability to set their domain as expired". I'm not sure what is the reason to have that beyond "more flexibility".